packages/webview/src/component/kube/KubeApplyYAML.svelte

```
@@ -0,0 +1,268 @@
+const availableContexts = states.stateAvailableContextsInfoUI;
+
+const contextNames = $derived(availableContexts.data?.contextNames ?? []);
+let selectedContextName = $derived(currentContext.data?.contextName ?? '');
+
+let unsubscribers: Unsubscriber[] = [];
+onMount(() => {
+  unsubscribers.push(currentContext.subscribe());
+  unsubscribers.push(availableContexts.subscribe());
+});
+onDestroy(() => {
+  unsubscribers.forEach(u => u());
```

Biome (2.4.7)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)